### ISSUE TITLE
Proper changelog

### ISSUE THREAD

**Issue by koenvervloesem** [rhasspy/rhasspy-services#3]
_Note: I'm creating this issue here because this applies to all Rhasspy services._

As part of a proper release process (see #2), we should also add a proper changelog to all Rhasspy services. See https://github.com/koenvervloesem/snipskit/blob/master/docs/source/CHANGELOG.rst for an example.

The way I did this with SnipsKit:

- Every time I made a non-trivial code change, I added a short description to the Unreleased section.
- When the time was right for a new release, I created a new section for the release and moved over these changes there.

There are other ways too, but I found that this way didn't give me much extra work and resulted in a consistent changelog.

Any thoughts about doing something like this for Rhasspy? You could then add this same changelog to the releases on GitHub, in your new release posts on the Rhasspy forum, in the Hass.io add-on and so on.

**Comment by synesthesiam** [comment]
I'm all for this, it just takes more discipline on my part 👍

**Comment by 0x326** [comment]
I've found the [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) guide to be a good example for my projects

**Comment by koenvervloesem** [comment]
@0x326 Yes, that's the one I used as a template for the SnipsKit example I linked :-) It has a very clear format and is easy to maintain.
